target/linux/ath79/config-5.4

```diff
@@ -90,6 +90,8 @@ CONFIG_GPIOLIB_IRQCHIP=y
 CONFIG_GPIO_74X164=y
 CONFIG_GPIO_ATH79=y
 CONFIG_GPIO_GENERIC=y
+CONFIG_GPIO_WATCHDOG=y
+CONFIG_GPIO_WATCHDOG_ARCH_INITCALL=y
```

Comment:
Can these symbols be added only for ath79/generic instead? Or is the size impact irrelevant?

Reply:
@adschm they are already enabled in `ath79/generic` and `ath79/mikrotik`.

Reply:
Yes, I've dropped this portion from the patchset. Don't ask me why I was so blind and didn't see that they were already enabled. Maybe I've used `make kernel_menuconfig` to check for the symbol (and kernel_menuconfig stuff doesn't take the config_default from generic).

Reply:
@ecsv
`make kernel_menuconfig CONFIG_TARGET=subtarget`
`make kernel_menuconfig CONFIG_TARGET=target`